dodo.py

```diff
@@ -5,13 +5,13 @@
 import re
 import time
 import os
+import sys
 from datetime import datetime
 from time import mktime
 
 
 DODO_FILE = os.path.join(os.getcwd(), 'DODO')
 VERSION = "0.99"
-username = os.path.split(os.path.expanduser('~'))[-1]
```

Comment:
this is to use the default system username. Can't remove it. So should be the case with 178-179 and 248-249. Can you explain the reason behind remove it?